src/theme/components/multiSelect.ts

```diff
@@ -26,8 +21,10 @@ const MultiSelect = {
     "menuButton",
     "menuContainer",
```

Comment:
The following are not used so please remove: "actionButtons", "menu", "menuButton", "menuContainer"